include/dxc/DXIL/DxilShaderFlags.h

```diff
@@ -227,17 +227,22 @@ class ShaderFlags {
       m_bDisableMathRefactoring : 1; //~D3D10_SB_GLOBAL_FLAG_REFACTORING_ALLOWED
   unsigned
       m_bEnableDoublePrecision : 1; // D3D11_SB_GLOBAL_FLAG_ENABLE_DOUBLE_PRECISION_FLOAT_OPS
+                                    // SHADER_FEATURE_DOUBLES
   unsigned
       m_bForceEarlyDepthStencil : 1; // D3D11_SB_GLOBAL_FLAG_FORCE_EARLY_DEPTH_STENCIL
 
   // Bit: 4
   unsigned
       m_bEnableRawAndStructuredBuffers : 1; // D3D11_SB_GLOBAL_FLAG_ENABLE_RAW_AND_STRUCTURED_BUFFERS
   unsigned
-      m_bLowPrecisionPresent : 1; // D3D11_1_SB_GLOBAL_FLAG_ENABLE_MINIMUM_PRECISION
+      m_bLowPrecisionPresent : 1; // AND ~m_bUseNativeLowPrecision => D3D11_1_SB_GLOBAL_FLAG_ENABLE_MINIMUM_PRECISION
+                                  // AND ~m_bUseNativeLowPrecision => SHADER_FEATURE_MINIMUM_PRECISION
+                                  // AND m_bUseNativeLowPrecision => SHADER_FEATURE_NATIVE_LOW_PRECISION
   unsigned
       m_bEnableDoubleExtensions : 1; // D3D11_1_SB_GLOBAL_FLAG_ENABLE_DOUBLE_EXTENSIONS
+                                     // SHADER_FEATURE_11_1_DOUBLE_EXTENSIONS
```

Comment:
As seen in [`DxilShaderFlags.cpp:L65`](https://github.com/microsoft/DirectXShaderCompiler/blob/5ff9cbc7cb83ab2d5f52255412f61ac3226c4a08/lib/DXIL/DxilShaderFlags.cpp#L65)